docs/docs/scheduled-triggers/recipes/daily-summary-email.mdx

```diff
@@ -138,10 +138,117 @@ Whenever a cron job is triggered, Hasura will send a request to the webhook URL
 simply going to send a `POST` request. Our webhook will parse the request, ensure the header is correct, and then query
 the database to determine which customers should receive a summary of their new notifications.
 
-Below, we've written an example of webhook in JavaScript. As we established earlier, this runs on port `4000`. If you're
+Below, we've written an example of webhook. As we established earlier, this runs on port `4000`. If you're
 attempting to run this locally, follow the instructions below. If you're running this in a hosted environment, use this
 code as a guide to write your own webhook.
 
+<Tabs
+  defaultValue="javascript"
+  values={[
+    { label: 'Python', value: 'python' },
+    { label: 'JavaScript', value: 'javascript' },
+  ]}
+>
+  <TabItem value="python">
+Make sure you have the necessary dependencies installed. You can use pip to install them:
+```bash
+pip install Flask secure-smtplib
+```
+
+<details>
+<summary>
+Then, create a new file called <code>index.py</code> and add the following code:
+</summary>
+
+```python
+from flask import Flask, request, jsonify
+import smtplib
```

Comment:
Let's bring in `SMTPException` as well. More context provided in the overall comment for the review.
```suggestion
import smtplib
from smtplib import SMTPException
```